internal/app/init/internal/rootfs/rootfs.go

```diff
@@ -76,6 +88,61 @@ func Prepare(s string, data *userdata.UserData) (err error) {
 	return nil
 }
 
+func generatePKI(data *userdata.UserData) (err error) {
+	log.Println("generating node identity PKI")
+	if data.IsBootstrap() {
+		log.Println("generating PKI from local key")
+		key, err := x509.NewKey()
```

Comment:
This entire branch is duplicate code of `internal/pkg/grpc/gen/gen.go`. I should make this DRY.